sonar-unit-tests.xsd

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Comment:
Yes, the .xsd file was included to validate the new XML file against. So in my case I used a external well-known xml tool (comes with the libxml-utils package in Linux) to do the actual validation. Can be dropped, certainly, doesn't affect the code.